sbi/utils/metrics.py

```diff
@@ -47,24 +82,20 @@ def c2st(
         X += noise_scale * torch.randn(X.shape)
         Y += noise_scale * torch.randn(Y.shape)
 
-    X = tensor2numpy(X)
-    Y = tensor2numpy(Y)
+    X = X.cpu().numpy()
+    Y = Y.cpu().numpy()
 
-    ndim = X.shape[1]
-
-    clf = MLPClassifier(
```

Comment:
we would want to keep this and construct the classifier class on top in an if-else on `c2st_classifier`, as commented above.

Reply:
this is honored with 5fd41c5